### PR TITLE
removed all instances of the var "funfact" and replaced w/ "bio". mig…

### PR DESCRIPTION
**Pull request by ktakattack**
…rated to DB.

Do not merge until Eyitayo reviews since he's the DB expert.

### CODE REVIEW

**Comment by tayoogunbiyi** [comment]
I’m no expert haha. Would review this.

File: models/user.js [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+    static associate(models) {
+      // define association here
+    }
+  };
+  User.init({
+    username: DataTypes.STRING,
+    discordId: DataTypes.STRING,
+    linkedin: DataTypes.STRING,
+    github: DataTypes.STRING,
+    bio: DataTypes.STRING,
```
Comment (tayoogunbiyi) [inline models/user.js:20]: This file is auto-generated so we don't want to manually edit it. We could, but however, the migration file should definitely not be auto edited. We should have a new migration file that reflects this specific DB structure change.

**Comment by tayoogunbiyi** [comment]
@ktakattack I added a new migration file so the MySQL knows about the column change. then I changed a place where the fun fact was still being referenced.

LGMT!

**Comment by ktakattack** [comment]
> @ktakattack I added a new migration file so the MySQL knows about the column change. then I changed a place where the fun fact was still being referenced.
> 
> LGTM!

Thank you for the assist!

**Pull request merged**